index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AASAN-UI</title>
+    <link rel="stylesheet" href="/css/alert.css"/> 
+    <link rel="stylesheet" href="/index.css" />
+</head>
+<body>
+    <div id="page1">
+        <header>
+        <div class = "header">
+            
+        <h1  class="head-logo-padd head-color" id="logo">AASAN-UI</h1>
+        <h3  class="head-padd">Home</h3>
+        <h3 class="head-padd head-color">
+        <a class="link" href="docs.html">Docs</a></h3>
+        <h3  class="head-padd head-color">
+        <a class="link" href="about.html">About</a></h3>
```

Comment:
Use lists here. Will be more semantic

Reply:
Okay I will take care of it from now on , I thought using just h3 would be fine.